.github/ISSUE_TEMPLATE/contributing.md

```diff
@@ -0,0 +1,171 @@
+
+2. Clone the forked repo to your local machine with one of the commands below. Be sure the `<your username>` text is replaced with your actual GitHub username, and the `<repo name>` with the actual repo name. You can also read the GitHub documentation on [cloning a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
+
+    ```bash
+    # If you have SSH set up with Git:
+    git clone git@github.com:<your username>/<repo name>.git
+    # Otherwise for HTTPS:
+    git clone https://github.com/<your username>/<repo name>.git
+
+    # An example:
+    git clone git@github.com:BryanF1nes/deals-for-devs.git
+    ```
+
+3. `cd` into the directory of your local clone, then set the upstream remote so you can keep your local clone synced with Deals for Devs original repo. The `<repo name>` below should be the same as the one you used when creating your local clone in the previous step.
+
+    ```bash
+    # If you have SSH set up with Git:
+    git remote add upstream git@github.com:<their username>/<repo name>.git
+    # Otherwise for HTTPS:
+    git remote add upstream https://github.com/<their username>/<repo name>.git
+
+    # An example:
+    git remote add upstream git@github.com:Learn-Build-Teach/deals-for-devs.git
+    ```
+
+### Working on an Issue
+
+Once you have the repo forked and cloned, and the upstream remote has been set, you can begin working on your issue:
+
+1. Create a new branch, replacing the `<your branch name>` with an actual branch name that briefly explains the purpose of the branch in some way:
+
+    ```bash
+    git checkout -b <your branch name>
+
+    # Some examples:
+    git checkout -b details_page
+    git checkout -b documentation_update
+    git checkout -b main_page_bug
+    ```
+
+2. Add commits as you work on your issue, replacing the `<your commit message>` text with your actual commit message:
+
+   ```bash
+   git commit -m "<your commit message>"
+
+   # An example:
+   git commit -m "Updated documentation"
+   ```
+
+3. Sync your work with the upstream remote every so often by running the following commands:
+
+    ```bash
+    # Fetching the most updated copy
+    git fetch upstream
+
+    # Merging the copy to your local main
+    git merge
+    ```
+
+4. Push your branch to your forked repo, replacing the `<your branch name>` with the branch you've been working on locally:
+
+    ```bash
+    git push origin <your branch name>
+
+    # An example:
+    git push origin details_page
+    ```
+
+### Opening a Pull Request
+
+1. After pushing your changes, go to your forked repo on GitHub and click the "Compare & pull request" button. If you have multiples of this button, be sure you click the one for the correct branch.
+   * If you don't see this button, you can click the branch dropdown menu and then select the branch you just pushed from your local clone:
```

Comment:
Can we add a blockquote here to call out that PRs should be made to the `dev1` branch?

Reply:
>  be made to the `dev1` branch?

Just to be clear this was a typo and should be `dev` branch?